### PR TITLE
Improve Site Tour UI

### PR DESCRIPTION
**Pull request by pedro-mendonca**
## Description

Improve the current UI:
- [x] Set a uniform color system, based on the `gp-primary-color`
- [x] Add i18n
- [x] Make the popover border radius concentric with the header radius
- [x] Use negative margins to compensate the spacing added by the popover pulse wrapper to avoid UI shifiting
- [x] Fix the UI shift from the pulse icon
- [x] Enqueue gp-tour and popover on Projects templates

## Testing Instructions

Add the below filter to test the Tours for Projects, Translation filters and editing, as below screenshots:
```
add_filter(
	'gp_tour',
	function( $tour ) {

		$new_tour = json_decode(
			'{
				"ui-intro" : [
					{
						"title": "GlotPress Introduction Tour",
						"color" : "#826eb4"
					},
					{
						"selector": ".gp-content p.actionlist a[href=\"/glotpress/projects/-new/\"]",
						"html": "Click here to create a new translation Project."
					},
					{
						"selector": ".gp-content p.actionlist a[href=\"/glotpress/languages/\"]",
						"html": "Click here to check your projects per language."
					},
					{
						"selector": "#project-actions-toggle",
						"html": "Click here to open your project actions."
					},
					{
						"reveal": "#project-actions-toggle",
						"selector": ".gp-content p.actionlist a[href=\"/glotpress/projects/test/import-originals/\"]",
						"html": "Click here to import originals"
					}
				],
				"filters-guide" : [
					{
						"title": "UI Introduction Tour",
						"color" : "#3939c7"
					},
					{
						"selector": ".revealing.filter",
						"html": "Click here to reveal the search field"
					},
					{
						"reveal": ".revealing.filter",
						"selector": ".filters-expanded input.is-primary",
						"html": "Click here to search"
					}
				],
				"translation-guide" : [
					{
						"title": "How to translate",
						"color" : "#f939c7"
					},
					{
						"selector": ".strings .original",
						"html": "This is the English text"
					},
					{
						"selector": ".strings .textareas",
						"html": "Here you can enter your translation"
					},
					{
						"selector": ".actions .is-primary",
						"html": "Submit your translation"
					}
				]
			}',
			true
		);

		return array_merge( $tour, $new_tour );
	}
);
```

## Screenshots

As published in the main PR [suggestion](https://github.com/GlotPress/GlotPress/pull/1632#issuecomment-1669664624):

imagem
imagem
imagem
imagem

### CODE REVIEW

**Comment by trymebytes** [comment]
@pedro-mendonca Please let me know if/when this is ready for review.

**Comment by pedro-mendonca** [comment]
@trymebytes ir is ready for review. Thanks :)

**Review by trymebytes — COMMENTED** [review]
Tested and works. In the scenario below, the border displaces the UI element (the text "Saturday"), is it the same for you?
tour-5

File: gp-templates/projects.php [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?php
 gp_title( __( 'Projects &lt; GlotPress', 'glotpress' ) );
 gp_breadcrumb( array( __( 'Projects', 'glotpress' ) ) );
+gp_enqueue_scripts( array( 'gp-jquery-webui-popover', 'gp-tour' ) );
```
Comment (ocean90) [inline gp-templates/projects.php:4]: `gp-tour` has `gp-jquery-webui-popover` as a dependency, why are both scripts needed here?

**Review by ocean90 — CHANGES_REQUESTED** [review]

**Comment by trymebytes** [comment]
I have cherrypicked some commits from here into the site-tour branch and I'll be closing this PR.

**Pull request closed without merging**